src/Cassandra/UdtMap.cs

```diff
@@ -83,7 +84,9 @@ public abstract class UdtMap
         protected readonly Dictionary<PropertyInfo, string> _propertyToFieldName;
         // ReSharper enable InconsistentNaming
         private Serializer _serializer;
-        protected const BindingFlags PropertyFlags = BindingFlags.FlattenHierarchy | BindingFlags.IgnoreCase | BindingFlags.Public | BindingFlags.Instance;
+        public const BindingFlags PropertyFlags = BindingFlags.FlattenHierarchy | BindingFlags.IgnoreCase | BindingFlags.Public | BindingFlags.Instance;
+
+        public TypeConverter TypeConverter { get; private set; }
```

Comment:
I don't think we should expose another conversion API, at least for now, before understanding if there is a need for it.

If there is a need for it in the future, it would be nice to expose a single one for both uses (either a new interface or the existent `TypeConverter`).